### ISSUE TITLE
Passing R to DENTIST

### ISSUE THREAD

**Issue by pcarbo** [Yves-CHEN/DENTIST#13]
Hi authors of DENTIST:
I would like to check consistency between my SNP-level summary statistics (z-scores) and an LD correlation matrix (R) obtained from a reference panel.
Is there a way to pass R as input to DENTIST without having to provide the individual-level data?

**Comment by Yves-CHEN** [comment]
Yes. It is a good idea to store LD matrix(R) and use R over the individual-level data. In fact, some efforts were made to store LD matrix in a storage-efficient manner, but the function has not been fully implemented yet. It will take quite some more to time to make it.

**Comment by pcarbo** [comment]
@Yves-CHEN Thanks for clarifying!

**Comment by MalteThodberg** [comment]
@Yves-CHEN has this feature been implemented yet?

**Comment by yanyul233** [comment]
Also eager to have this feature
